waspc/src/Wasp/Psl/Parser/Model.hs

```diff
@@ -1,6 +1,8 @@
 module Wasp.Psl.Parser.Model
   ( parseBody,
     model,
+    -- Used by the `view` and `type` parsers.
```

Comment:
I would remove this maybe -> it will invetiable become outdated at some point, and if somebody wants to check for usage, they can use their editor.

Reply:
Ok, good point 👍